examples/vite/src/application.tsx

```diff
@@ -186,8 +187,8 @@ export function App() {
           title={{
             value: "Where do you live?",
             asHeading: {
-              size: 'md',
-              as: 'h3',
+              size: "md",
```

Comment:
I believe our lint configs aren’t aligned, it would be helpful to ensure we’re all using the same linting configuration to avoid these small unintentional changes across the files.

Reply:
The issue here was that we didn't have the prettier config set up for the vite example, I added that now and it should apply the same rules for all of us